packages/cli/src/tools/packageManager.ts

```diff
@@ -41,9 +41,21 @@ function configurePackageManager(
   action: 'init' | 'install' | 'installDev' | 'installAll' | 'uninstall',
   options: Options,
 ) {
-  let pm: PackageManager = shouldUseYarn(options) ? 'yarn' : 'npm';
+  let yarnAvailable = shouldUseYarn(options);
+  let bunAvailable = shouldUseBun(options);
+
+  let pm: PackageManager = 'npm';
+
   if (options.packageManager === 'bun') {
-    pm = shouldUseBun(options) ? 'bun' : 'npm';
+    if (bunAvailable) {
+      pm = 'bun';
+    } else {
+      pm = yarnAvailable ? 'yarn' : 'npm';
+    }
+  }
```

Comment:
I somehow feel better about not mixing if/else with ternary
```suggestion
    if (bunAvailable) {
      pm = 'bun';
    } else if (yarnAvailable) {
      pm = 'yarn';
    } else {
      pm = 'npm';
    }
  }
```
switch would make it even more readable maybe. Your call :)